webextensions/api/privacy.json

```diff
@@ -62,7 +62,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": true
+                  "version_added": "≤90"
```

Comment:
This is a version range based on the browser-compat-data commit.

Reply:
As noted in Rob's review, despite it being added to the compatability data in #8830, in reality it does not exist on Chromium browsers.